ext/sqlite3/sqlite3.stub.php

```diff
@@ -10,131 +10,131 @@ class SQLite3
+    /** @tentative-return-type */
+    public function enableExtendedResultCodes(bool $enable = true): bool {}
 
-    /** @return bool */
-    public function setAuthorizer(?callable $callback) {}
+    /** @tentative-return-type */
+    public function setAuthorizer(?callable $callback): bool {}
 }
 
 class SQLite3Stmt
 {
     private function __construct(SQLite3 $sqlite3, string $query) {}
 
-    /** @return bool */
-    public function bindParam(string|int $param, mixed &$var, int $type = SQLITE3_TEXT) {}
+    /** @tentative-return-type */
+    public function bindParam(string|int $param, mixed &$var, int $type = SQLITE3_TEXT): bool {}
 
-    /** @return bool */
-    public function bindValue(string|int $param, mixed $value, int $type = SQLITE3_TEXT) {}
+    /** @tentative-return-type */
+    public function bindValue(string|int $param, mixed $value, int $type = SQLITE3_TEXT): bool {}
 
-    /** @return bool */
-    public function clear() {}
+    /** @tentative-return-type */
+    public function clear(): bool {}
 
-    /** @return bool */
-    public function close() {}
+    /** @tentative-return-type */
+    public function close(): bool {}
 
-    /** @return SQLite3Result|false */
-    public function execute() {}
+    /** @tentative-return-type */
+    public function execute(): SQLite3Result|false {}
 
-    /** @return string|false */
-    public function getSQL(bool $expand = false) {}
+    /** @tentative-return-type */
+    public function getSQL(bool $expand = false): string|false {}
 
-    /** @return int */
-    public function paramCount() {}
+    /** @tentative-return-type */
+    public function paramCount(): int {}
 
-    /** @return bool */
-    public function readOnly() {}
+    /** @tentative-return-type */
+    public function readOnly(): bool {}
 
-    /** @return bool */
-    public function reset() {}
+    /** @tentative-return-type */
+    public function reset(): bool {}
 }
 
 class SQLite3Result
 {
     private function __construct() {}
 
-    /** @return int */
-    public function numColumns() {}
+    /** @tentative-return-type */
+    public function numColumns(): int {}
 
-    /** @return string|false */
-    public function columnName(int $column) {}
+    /** @tentative-return-type */
+    public function columnName(int $column): string|false {}
 
-    /** @return int|false */
-    public function columnType(int $column) {}
+    /** @tentative-return-type */
+    public function columnType(int $column): int|false {}
 
-    /** @return array|false */
-    public function fetchArray(int $mode = SQLITE3_BOTH) {}
+    /** @tentative-return-type */
+    public function fetchArray(int $mode = SQLITE3_BOTH): array|false {}
 
-    /** @return bool */
-    public function reset() {}
+    /** @tentative-return-type */
+    public function reset(): bool {}
 
-    /** @return bool */
-    public function finalize() {}
+    /** @tentative-return-type */
+    public function finalize(): bool {}
```

Comment:
This is another "always true" close method.

Reply:
Nice catch! I'm going to get rid of the tentative return type then.